src/functions/template-tags/html.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * HTML functions (template-tags) for use in WordPress templates.
+ * HTML functions ( template-tags ) for use in WordPress templates.
```

Comment:
I think some find/replace process didn't distinguish between comments and code 😄

Reply:
nope doh... 🤦🏼